docs/api/v1.24.md

```diff
@@ -1659,6 +1659,7 @@ Build an image from a Dockerfile
 **Example request**:
 
     POST /v1.24/build HTTP/1.1
+    Content-Type: application/tar
```

Comment:
I think this has to be `x-tar`, because that's what we use everywhere. There is no official mime-type for tar, so both `tar` and `x-tar` are used, but all our other examples use `x-tar`

Can you also check the older versions of the documentation?